xml/dm_registers.xml

```diff
@@ -357,7 +357,7 @@
             (e.g.\ while executing the Program Buffer).
 
             4 (halt/resume): The abstract command couldn't execute because the
-            hart wasn't in the required state (running/halted).
+            hart wasn't in the required state (running/halted), or unavailable.
```

Comment:
I think we should just make this its own error type, or do what you did before and rename this error type. 

What if a hart becomes unavailable during the course of this? Didn't we have another PR that adds a new error type for that?

Reply:
I don't see a need for a separate error code for this case.  Clearly the hart is in the wrong state when the abstract command is attempted.  I don't have a strong feeling about the name.

Reply:
@mwachs5 How important is changing the error code to you? I think that's the only thing that's preventing this PR from being approved.